store/tikv/2pc.go

```diff
@@ -46,11 +46,12 @@ import (
 type twoPhaseCommitAction interface {
 	handleSingleBatch(*twoPhaseCommitter, *Backoffer, batchMutations) error
 	tiKVTxnRegionsNumHistogram() prometheus.Observer
+	inRetry() bool
```

Comment:
inRetryCommit to be exact?
Maybe we don't need to add this method to the interface.